site/cypress/e2e/Cypress-Gradeable/rainbow_grading.spec.js

```diff
@@ -4,18 +4,166 @@ skipOn(Cypress.env('run_area') === 'CI', () => {
+            // TODO: test drag/drop functionality
+            cy.get('[data-testid="gradeables"]').should('be.visible');
+            cy.get('[data-testid="buckets-used-list"]').should('be.visible');
+            cy.get('[data-testid="buckets-available-list"]').should('be.visible');
+            cy.get('[data-testid="gradeable-config"]').should('be.visible');
+
+            // Ensure textboxes have correct initial values and can be modified
+            checkTextbox('[data-testid="cust-messages-textarea"]', '', 'message');
+            checkTextbox('[data-testid="benchmark_lowest_a-"]', '0.9', '0.8');
+            checkTextbox('[data-testid="benchmark_lowest_b-"]', '0.8', '0.7');
+            checkTextbox('[data-testid="benchmark_lowest_c-"]', '0.7', '0.6');
+            checkTextbox('[data-testid="benchmark_lowest_d"]', '0.6', '0.5');
+            /* TODO: once uncaught saving issue is fixed, include the below tests for section labels
+            checkTextbox('[data-testid="section_and_labels_1"]', '1', 'TA 1');
+            checkTextbox('[data-testid="section_and_labels_2"]', '2', 'TA 2');
+            checkTextbox('[data-testid="section_and_labels_3"]', '3', 'TA 3');
+            checkTextbox('[data-testid="section_and_labels_4"]', '4', 'TA 4');
+            checkTextbox('[data-testid="section_and_labels_5"]', '5', 'TA 5');
+            checkTextbox('[data-testid="section_and_labels_6"]', '6', 'TA 6');
+            checkTextbox('[data-testid="section_and_labels_7"]', '7', 'TA 7');
+            checkTextbox('[data-testid="section_and_labels_8"]', '8', 'TA 8');
+            checkTextbox('[data-testid="section_and_labels_9"]', '9', 'TA 9');
+            checkTextbox('[data-testid="section_and_labels_10"]', '10', 'TA 10');
+             */
+            checkTextbox('[data-testid="cutoff_A"]', '93', '87');
+            checkTextbox('[data-testid="cutoff_A-"]', '90', '80');
+            checkTextbox('[data-testid="cutoff_B+"]', '87', '77');
+            checkTextbox('[data-testid="cutoff_B"]', '83', '73');
+            checkTextbox('[data-testid="cutoff_B-"]', '80', '70');
+            checkTextbox('[data-testid="cutoff_C+"]', '77', '67');
+            checkTextbox('[data-testid="cutoff_C"]', '73', '63');
+            checkTextbox('[data-testid="cutoff_C-"]', '70', '60');
+            checkTextbox('[data-testid="cutoff_D+"]', '67', '55');
+            checkTextbox('[data-testid="cutoff_D"]', '60', '50');
+
+            // Ensure tables can be added to and removed from
+            // TODO: add checks for other tables once features are implemented
+            cy.get('[data-testid="manual-grading-user-id"]').type('adamsg');
+            cy.get('[data-testid="manual-grading-grade"]').select(1);
+            cy.get('[data-testid="manual-grading-note"]').type('MESSAGE');
+            cy.get('[data-testid="manual-grading-submit"]').click();
+
+            cy.get('[data-testid="manual-grading-table-body"] > tr > td').eq(0).should('contain', 'adamsg');
```

Comment:
Using an alias here might make the code more readable. For example 
`cy.get('[data-testid="manual-grading-table-body"] > tr > td').as('table-elements');`
`
cy.get(@table-elements).eq(1).should('contain', 'A');
`
`
cy.get(@table-elements).eq(2).should('contain', 'MESSAGE');
`

Reply:
Changed to alias in [9663872](https://github.com/Submitty/Submitty/pull/10726/commits/9663872cd37481058f46075716acc09ed874b4ad)